go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/mattn/go-isatty v0.0.17
 	github.com/mongodb-forks/digest v1.0.4
 	github.com/mongodb-labs/cobra2snooty v0.12.2
-	github.com/mongodb/mongodb-atlas-kubernetes v1.5.1
```

Comment:
We should not update this to a not released version. Do we need to update this library to update these tests? if yes, we can wait for the kub release to merge this PR.

Reply:
I will create a new tag to highlight that this PR is waiting for the kub release

Reply:
Yeah. Release 1.6.0 is planned for this Friday.